edison/config.py

```diff
@@ -0,0 +1,36 @@
+import inspect
+import sys
+
+
+def get_config_object(env_keyword: str):
+    """
+    Returns the the desired config class path.
+
+    The function iterates through a dictionary returned by inspect.
+    The dictionary contains details about all of the file members.
+    Its key is the name of the member and value is obj which contains all the details about the member.
+    The desired config path is being picked by the ENV_KEYWORD field defined in the config class.
+
+    Parameters:
+    env_keyword (str): Should be equals to one of the config classes ENV_KEYWORD field.
+
+    Returns:
+    str: module_name.class_name, which is the full path of the config class.
+    """
+    for name, obj in inspect.getmembers(sys.modules[__name__]):
+        if inspect.isclass(Config) and obj.ENV_KEYWORD == env_keyword:
+            return ".".join([obj.__module__, name])
```

Comment:
This doesn't check if the member returned in the loop is a Config, it checks if Config itself is a class.
If Config is not defined, this will cause an undefined object error.
If Config will always be a class, then this code does nothing.
I think what you want to code is:
if issubclass(obj, Config) ...

Reply:
ack and fixed :)